js/turtleactions/MeterActions.js

```diff
@@ -373,6 +379,7 @@ function setupMeterActions() {
         }
 
         static getBeatFactor(turtle) {
+            // eslint-disable-next-line max-len
```

Comment:
pls don't disable rules unless extraordinary circumstances. This one exceeds 100 characters, hence the warning.